edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/Constants.cs

```diff
@@ -28,6 +28,6 @@ public class Constants
         public const string ServiceApiIdHeaderKey = "x-ms-edge-moduleId";
         public const string OriginEdgeHeaderKey = "x-ms-edge-origin";
 
-        public static readonly Version ConfigSchemaVersion = new Version("1.1.0");
+        public static readonly Version ConfigSchemaVersion = new Version("1.2.0");
```

Comment:
So this PR cannot be merged until the new twin schema is rolled out?

Reply:
Good question, I'm not sure. I thought that this is max supported version, and all previous version would still work. @varunpuranik, @ancaantochi, @richma-ms do you have any input on this?

Reply:
What do you mean by new twin schema is rolled out? 
It would be the other way round. Also, Vadim is right, this would be the max supported version
There is a version matching item on @richma-ms, but that may/may not come before we release this.